src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.Dao;
+import core.basesyntax.dao.DaoImpl;
+import core.basesyntax.service.Convert;
+import core.basesyntax.service.ConvertImpl;
+import core.basesyntax.service.ReadData;
+import core.basesyntax.service.ReadDataImpl;
+import core.basesyntax.service.StartingBalance;
+import core.basesyntax.service.StartingBalanceImpl;
+import core.basesyntax.service.Writer;
+import core.basesyntax.service.WriterImpl;
+import core.basesyntax.service.strategy.Activities;
+import core.basesyntax.service.strategy.Purchase;
+import core.basesyntax.service.strategy.Return;
+import core.basesyntax.service.strategy.Supply;
+
+public class Main {
+    private static final Activities sup = new Supply();
+    private static final Activities ret = new Return();
+    private static final Activities pur = new Purchase();
+    private static final ReadData readData = new ReadDataImpl();
+    private static final Convert convert = new ConvertImpl();
+    private static final Writer writer = new WriterImpl();
+    private static final Dao transferDao = new DaoImpl();
+    private static final StartingBalance startingBalance = new StartingBalanceImpl();

```

Comment:
bad naming
developers spend much more time reading others' code than writing their own
a second you save by not typing out a few extra characters are not worth it in comparison with enhanced readability 

You lack a strategy in your implementation
Remove this form main, create a separate class, with a hashMap field, key is operation/activity, value - OperationHandler (your Activites)